test/src/e2e.long/mempoolMinfee.test.ts

```diff
@@ -160,22 +163,22 @@ describe("MemPoolMinFees", async function() {
         await Promise.all(nodeArray.map(node => node.waitBlockNumber(4)));
         const expectedTrues = await Promise.all(
             nodeArray.map(node =>
-                node.sdk.rpc.chain.containsTransaction(
-                    txShouldBeAccepted.hash()
-                )
+                node.rpc.chain.containsTransaction({
+                    transactionHash: `0x${txShouldBeAccepted.hash().toString()}`
+                })
             )
         );
         const expectedFalses = await Promise.all(
             nodeArray.map(node =>
-                node.sdk.rpc.chain.containsTransaction(
-                    txShouldBeRejected.hash()
-                )
+                node.rpc.chain.containsTransaction({
+                    transactionHash: `0x${txShouldBeRejected.hash().toString()}`
+                })
             )
         );
 
         expectedTrues.map(val => expect(val).to.be.true);
         expectedFalses.map(val => expect(val).to.be.false);
 
         await nonValNode.clean();
-    });
+    }).timeout(10_000);
```

Comment:
Why did you add it in this commit?
If you found that this test failed often because of timeout, please create a new commit for it.